src/ICSharpCode.SharpZipLib/Zip/ZipOutputStream.cs

```diff
@@ -257,8 +257,11 @@ public void PutNextEntry(ZipEntry entry)
 						if (!CanPatchEntries)
 						{
 							// Can't patch entries so storing is not possible.
-							method = CompressionMethod.Deflated;
-							compressionLevel = 0;
+							// Update: support for GeneralBitFlags.Descriptor on non-deflate methods
+							//         have been present in PKZIP for ~20 years now
+							//method = CompressionMethod.Deflated;
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/piksel/SharpZipLib/pullRequest?prid=5745485)